spec/bookshelf-spec.ts

```diff
@@ -633,11 +633,13 @@ describe('Bookshelf relations', () => {
     bookshelf.knex.destroy(done);
   });
 
-  it('should add relationships object', () => {
+  fit('should add relationships object', () => {
```

Comment:
It stayed there from some debugging I had to do, and I think you're being a little over the top with that "wrong in so many levels" statement.